OpenRA.Game/Settings.cs

```diff
@@ -300,6 +302,52 @@ public void Save()
 			root.WriteToFile(settingsFile);
 		}
 
+		public static string SanitizedName(string dirty)
+		{
+			// reserved characters for MiniYAML and JSON
+			var disallowedChars = new char[] { '#', '@', ':', '\n', '\t', '[', ']', '{', '}', '"', '`' };
+
+			var clean = dirty;
+			if (string.IsNullOrEmpty(dirty))
+				clean = "";
```

Comment:
This will be replaced by `Newbie` and `OpenRA Game` (default server name) in the functions below.